source/Calamari.GoogleCloudScripting/GoogleCloudContextScriptWrapper.cs

```diff
@@ -222,6 +205,129 @@ CommandResult ExecuteCommand(params string[] arguments)
                 return result;
             }
 
+            bool SetGcloudExecutable()
```

Comment:
The convention for returning a boolean to indicate success would be `TrySetGcloudExecutable`